client/my-sites/site-settings/media-settings.jsx

```diff
@@ -17,111 +18,182 @@ import CompactFormToggle from 'components/forms/form-toggle/compact';
+	PLAN_JETPACK_PREMIUM_MONTHLY,
+	FEATURE_VIDEO_UPLOADS_JETPACK_PREMIUM,
+} from 'lib/plans/constants';
+import {
 	isJetpackModuleActive,
 	isJetpackModuleUnavailableInDevelopmentMode,
 	isJetpackSiteInDevelopmentMode
 } from 'state/selectors';
 import { getSelectedSiteId } from 'state/ui/selectors';
+import { getSitePlanSlug } from 'state/sites/selectors';
 import { updateSettings } from 'state/jetpack/settings/actions';
 import QueryJetpackConnection from 'components/data/query-jetpack-connection';
 
-const MediaSettings = ( {
-	fields,
-	handleAutosavingToggle,
-	onChangeField,
-	isRequestingSettings,
-	isSavingSettings,
-	siteId,
-	carouselActive,
-	photonModuleUnavailable,
-	selectedSiteId,
-	translate
-} ) => {
-	const labelClassName = isSavingSettings || ! carouselActive ? 'is-disabled' : null;
+class MediaSettings extends Component {
+	static propTypes = {
+		carouselActive: PropTypes.bool.isRequired,
+		fields: PropTypes.object,
+		handleAutosavingToggle: PropTypes.func.isRequired,
+		isRequestingSettings: PropTypes.bool,
+		isSavingSettings: PropTypes.bool,
+		isVideoPressAvailable: PropTypes.bool,
+		onChangeField: PropTypes.func.isRequired,
+		siteId: PropTypes.number.isRequired,
+	};
 
-	return (
-		<Card className="media-settings site-settings site-settings__module-settings">
-			<QueryJetpackConnection siteId={ selectedSiteId } />
+	renderVideoSettings() {
+		const {
+			isRequestingSettings,
+			isSavingSettings,
+			isVideoPressAvailable,
+			siteId,
+			translate,
+		} = this.props;
+		const isRequestingOrSaving = isRequestingSettings || isSavingSettings;
 
-			<FormFieldset>
-				<div className="media-settings__info-link-container site-settings__info-link-container">
-					<InfoPopover position="left">
-						<ExternalLink target="_blank" icon={ true } href="https://jetpack.com/support/photon" >
-							{ translate( 'Learn more about Photon.' ) }
-						</ExternalLink>
-					</InfoPopover>
-				</div>
-				<JetpackModuleToggle
-					siteId={ siteId }
-					moduleSlug="photon"
-					label={ translate( 'Speed up your images and photos with Photon' ) }
-					description={ translate( 'Must be enabled to use tiled galleries.' ) }
-					disabled={ isRequestingSettings || isSavingSettings || photonModuleUnavailable }
-					/>
-			</FormFieldset>
-			<FormFieldset className="media-settings__formfieldset has-divider is-top-only">
-				<div className="media-settings__info-link-container site-settings__info-link-container">
+		return isVideoPressAvailable
+			? <FormFieldset className="site-settings__formfieldset has-divider is-top-only">
+				<div className="site-settings__info-link-container">
 					<InfoPopover position="left">
-						<ExternalLink target="_blank" icon={ true } href="https://jetpack.com/support/carousel" >
-							{ translate( 'Learn more about Carousel.' ) }
+						<ExternalLink target="_blank" icon href="https://jetpack.com/support/videopress/" >
+							{ translate( 'Learn more about VideoPress.' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **33** times:
`translate( 'Learn more about CSS' )` *ES Score: 7*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).